PowerShell/BuildingBlocks/SDKOperations/RetrieveEntities.ps1

```diff
@@ -1,11 +1,16 @@
-# Initialize the context
-# Create a customer scoped ticket
-.\Init.ps1 -Customer
+# Authenticate
+$creds = Get-Credential -Message "Enter BitTitan credentials"
+$ticket = Get-BT_Ticket -Credentials $creds -ServiceType BitTitan
+
+# Retrieve a customer and get a ticket for it
+$customerId = [GUID](Read-Host -Prompt 'Customer ID') 
+$customer = Get-BT_Customer -Ticket $ticket -Id $customerId
+$customerTicket = Get-BT_Ticket -Ticket $ticket -OrganizationId $customer.OrganizationId
 
 # Here are 3 common ways to retrieve entities
 # 1. Retrieve all the endpoints under a customer and process them one by one
 # Use -RetrieveAll and piping with ForEach 
-Get-BT_Endpoint -Ticket $mspc.CustomerTicket -RetrieveAll | ForEach {
+Get-BT_Endpoint -Ticket $customerTicket -RetrieveAll | ForEach {
```

Comment:
Q: by default I believe PowerShell does not filter out soft-deleted entities. If the caller needs to only get non-deleted records, they need to manually pass "-IsDeleted $false". Does it makes sense to include this in the example?

Reply:
Makes sense, added